src/pages/Scheduling/components/CreateScheduleTemplateSheet.tsx

```diff
@@ -431,7 +431,7 @@ export default function CreateScheduleTemplateSheet({
                         )}
                       /> */}
 
-                      <div className="flex items-center gap-4 col-span-2 md:col-span-1">
+                      <div className="flex items-center gap-1 sm:gap-4 col-span-2 md:col-span-1">
```

Comment:
Does this look right?

<img width="480" alt="image" src="https://github.com/user-attachments/assets/f615347a-7ad6-47a1-8439-f95adbd23744" />

Let's adjust the layout here as well, so that start/end time etc are all on their own lines for smaller devices.

You would need to adjust the icon are well - maybe ArrowDownIcon, test and see if that makes sense.